control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -46,11 +46,14 @@ var (
 			oasVolumeKonnectivityProxyCert().Name: "/etc/konnectivity-proxy-tls",
 		},
 	}
-	openShiftAPIServerLabels = map[string]string{
+)
+
+func openShiftAPIServerLabels() map[string]string {
```

Comment:
@alvaroaleman caught another one...

Reply:
While doing this I also noticed openShiftOAuthAPIServerLabels needs the function treatment, will do that in a PR for the oauth disruption budget.